infrastructure/docker/traffic_ops/Dockerfile

```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
 ############################################################
 # Dockerfile to build Traffic Ops 1.6.0 container images
 # Based on CentOS 7.2
```

Comment:
can we get rid of the TO version?   Maybe say 2.x?

Reply:
Fixed.

I removed the version entirely, since we'd forget to update it with 3.x, and it's clear from the context of the git checkout which version it is.

Reply:
+1